setup.py

```diff
@@ -74,24 +74,40 @@ def get_version(version_file):
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11",
-        "Programming Language :: Python :: Implementation :: CPython",
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="posture and exposure source",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
+    package_data={
+        "pe_source": [
+            "data/*",
+            "data/shodan/*",
+            "data/sixgill/*",
+            "data/dnsmonitor/*",
+            "data/pe_db/*",
```

Comment:
These should be sorted alphabetically:
```suggestion
            "data/*",
            "data/dnsmonitor/*",
            "data/pe_db/*",
            "data/shodan/*",
            "data/sixgill/*",
```